ci/experimental/docker/connector_build/scripts/entrypoint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
Instead of copying this file even more times, could we set soft links instead?
Could you look whether this is an option in git?